codeguru_profiler_agent/sdk_reporter/sdk_reporter.py

```diff
@@ -90,11 +97,64 @@ def report(self, profile):
             )
             logger.info("Reported profile successfully")
             return True
+        except ClientError as error:
```

Comment:
Could you please update the documentation of this functions that this creates a PG in the case of using Lamba layers if it throws ResourceNotFoundException? I would be clearer when reading the code.

Reply:
done!